### PR TITLE
Random cleanups

### PR DESCRIPTION
**Pull request by relrelb**
Various cleanups across the codebase, mostly config files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-dist/
-pkg/
-web/packages/core/docs
-web/packages/core/tsd
-*.swd
 
-# NPM
-_cacache/
-_logs/
-anonymous-cli-metrics.json
```
Comment (Herschel) [inline .gitignore:13]: Why remove these?

Comment (Herschel) [inline .gitignore:13]: Why remove these?

File: web/packages/core/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/pkg/
+/docs/
+/tsd/
```
Comment (relrelb) [inline web/packages/core/.gitignore:3]: I'm not sure what `tsd` is. Maybe something related to TypeScript?
If it's not relevant anymore, we can remove it.

Comment (Herschel) [reply web/packages/core/.gitignore:3]: I guess it's typescript definitions, but not sure how/if it was ever generated.

**Review by Herschel — APPROVED** [review]

**Pull request merged**